sources/Renderer/Vulkan/VKRenderSystem.cpp

```diff
@@ -445,11 +460,15 @@ void VKRenderSystem::WriteTexture(Texture& texture, const TextureRegion& texture
     /* Check if image data must be converted */
     DynamicByteArray intermediateData;
 
+    std::uint32_t srcRowStride = srcImageView.rowStride > 0 ? srcImageView.rowStride : extent.width * bytesPerPixel;
```

Comment:
Same question here as above. Let me know if you think a `max()` would make sense here.